apps/web/src/pages/Pools/PoolAdvertisementPage/components/LanguageRequirementDialog.tsx

```diff
@@ -0,0 +1,461 @@
+                  {
+                    defaultMessage:
+                      "Try out the <link>self-assessment</link> tests to get a sense of your current level.",
+                    id: "x20cWU",
+                    description:
+                      "Description for bilingual positions in language requirements dialog",
+                  },
+                  {
+                    link: (chunks: ReactNode) =>
+                      selfAssessmentLink(locale, chunks),
+                  },
+                )}
+              </p>
+            </div>
+          </div>
+          <div>
+            <Heading
+              level={headingLevel}
+              size="h6"
+              data-h2-margin-top="base(0)"
+            >
+              {intl.formatMessage({
+                defaultMessage: "Second language proficiency",
+                id: "fSeYdG",
+                description: "Sub-heading for language requirements dialog",
+              })}
+            </Heading>
+            <div
+              data-h2-display="base(grid)"
+              data-h2-gap="base(x.5)"
+              data-h2-margin-bottom="base(x.5)"
+            >
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "The Government of Canada uses letter codes to describe second language proficiency. There are three proficiency levels for general qualifications",
+                  id: "sgt8f7",
+                  description:
+                    "Language proficiency list for language requirements dialog",
+                }) + intl.formatMessage(commonMessages.dividingColon)}
+              </p>
+              <ul data-h2-display="base(grid)" data-h2-gap="base(x.5)">
+                <li>
+                  {intl.formatMessage({
+                    defaultMessage: `"A" is the lowest level of bilingual ability and indicates a beginner level.`,
+                    id: "qD0lyZ",
+                    description:
+                      "Proficiency level on language requirements dialog",
+                  })}
+                </li>
+                <li>
+                  {intl.formatMessage({
+                    defaultMessage: `"B" indicates an intermediate level.`,
+                    id: "DoZebI",
+                    description:
+                      "Proficiency level on language requirements dialog",
+                  })}
+                </li>
+                <li>
+                  {intl.formatMessage({
+                    defaultMessage: `"C" is the highest and indicates an advanced level.`,
+                    id: "e9jncy",
+                    description:
+                      "Proficiency level on language requirements dialog",
+                  })}
+                </li>
+              </ul>
+            </div>
+            <div
+              data-h2-display="base(grid)"
+              data-h2-gap="base(x.5)"
+              data-h2-margin-bottom="base(x.5)"
+            >
```

Comment:
Same as previous comment. I think we should rely more on individual element spacing. You could create a tiny  `<Text>` element to avoid repeating yourself.

Reply:
Fixed by [7957bec](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10628/commits/7957bec46d036ea078394010540617660e66b9df)